src/traits.rs

```diff
@@ -187,26 +187,47 @@ pub trait CheckedSub<Rhs = Self>: Sized {
     fn checked_sub(&self, rhs: Rhs) -> CtOption<Self>;
 }
 
-/// Concatenate two numbers into a "wide" twice-width value, using the `rhs`
+/// Concatenate two numbers into a "wide" combined-width value, using the `lo`
 /// value as the least significant value.
-pub trait Concat<Rhs = Self> {
+pub trait ConcatMixed<Lo: ?Sized = Self> {
```

Comment:
I think it would make sense to define this below `Concat` (and same for `Split`/`SplitMixed`)

Reply:
Thanks, updated